chain/client/src/test_utils/setup.rs

```diff
@@ -162,6 +162,11 @@ pub fn setup(
         chain_genesis,
         epoch_manager,
         shard_tracker,
+        Arc::new(RwLock::new(SyncAdapter::new(
```

Comment:
Extract the expression into a variable. As it is now, it breaks the flow of reading arguments of a function.